providers/openfeature-provider-flagd/src/openfeature/contrib/provider/flagd/config.py

```diff
@@ -1,23 +1,27 @@
 import os
 import typing
 
+T = typing.TypeVar("T")
+
 
 def str_to_bool(val: str) -> bool:
     return val.lower() == "true"
 
 
-def env_or_default(env_var, default, cast=None):
+def env_or_default(
+    env_var: str, default: T, cast: typing.Optional[typing.Callable] = None
```

Comment:
We could type the Callable to avoid the type error, but this is an internal function so it's fine for me either way.

Reply:
yeah, sure. can do 🙂